templates/SilverStripe/View/Shortcodes/EmbedShortcodeProvider_link.ss

```diff
@@ -0,0 +1,6 @@
+<a
+    href="{$Href}"
+    <% loop $Attributes %>{$Name}="{$Value}"<% end_loop %>
```

Comment:
Is there missing whitespace here? I don't think it matters though.

Reply:
Does this need to be `$Value.ATT`?

Reply:
Good spot, I’ll fix that